creator-node/test/transcodeAndSegment.test.js

```diff
@@ -0,0 +1,96 @@
+const request = require('supertest')
+const sinon = require('sinon')
+const uuid = require('uuid')
+
+const BlacklistManager = require('../src/blacklistManager')
+
+const { getApp } = require('./lib/app')
+const { getIPFSMock } = require('./lib/ipfsMock')
+const { getLibsMock } = require('./lib/libsMock')
+
+describe('test transcode_and_segment route', function () {
+  let app, server, testUuid
+
+  before(function () {
+    testUuid = uuid.v4()
+  })
+
+  beforeEach(async function () {
+    const appInfo = await getApp(
+      /* ipfsMock */ getIPFSMock(),
+      /* libsMock */ getLibsMock(),
+      BlacklistManager,
+      /* ipfsLatestMock */ getIPFSMock(true),
+      /* setMockFn */ null,
+      1 /* spId */
+    )
+
+    app = appInfo.app
+    server = appInfo.server
+  })
+
+  afterEach(async () => {
+    sinon.restore()
+    await server.close()
+  })
+
+  it('if uuid/file name/file type is not passed in, return 400', async function () {
+    await request(app)
+      .get('/transcode_and_segment')
+      .query({
+        uuid: testUuid,
+        fileName: undefined,
+        fileType: 'transcode',
+        // To pass ensureValidSPMiddleware
```

Comment:
is this comment relevant to the test case?

Reply:
yeah, there's a middleware in this route. I thought this comment would make it clear as to why these not super relevant query strings were passed in